Source/WebCore/platform/graphics/cocoa/MediaPlaybackTargetContext.serialization.in

```diff
@@ -31,8 +33,22 @@ enum class WebCore::MediaPlaybackTargetContextMockState : uint8_t {
 enum class WebCore::MediaPlaybackTargetContextType : uint8_t {
     None,
     AVOutputContext,
-    SerializedAVOutputContext,
     Mock,
 };
 
+[Nested] struct WebCore::MediaPlaybackTargetContext::NonPlatformData {
+    String mockDeviceName;
+    WebCore::MediaPlaybackTargetContextMockState mockState;
+}
+
+[Nested] struct WebCore::MediaPlaybackTargetContext::PlatformData {
+    RetainPtr<AVOutputContext> outputContext;
+}
+
+class WebCore::MediaPlaybackTargetContext {
+    bool encodingRequiresPlatformData();
```

Comment:
Encoding this boolean in the IPC doesn't make sense to me. The recipient can just check if the variant contains platform data or not.